### PR TITLE
Align Title And Prompts left pane + Add code linter and formatter

### PR DESCRIPTION
**Pull request by DoroNahari**

### CODE REVIEW

File: frontend/src/components/Navigation.vue [hunk 1]
```diff
@@ -52,13 +52,13 @@ div.v-stepper div.v-stepper__step {
 }
 
 .v-stepper__step--complete.v-stepper__step {
-  background: var(--vscode-editorCodeLens-foreground, #252426);
+  background: var(--vscode-menu-background, #252426);
 }
 .v-stepper__step--active.v-stepper__step {
   background: var(--vscode-titleBar-activeBackground, #383838);
```
Comment (tomer-epstein) [inline frontend/src/components/Navigation.vue]: --vscode-titleBar-activeBackground not supported in theia yet. We should chose another common property.

File: frontend/package.json [hunk 2]
```diff
@@ -11,15 +11,20 @@
...
     "bootstrap": "^4.3.1",
     "bootstrap-vue": "^2.0.2",
+    "core-js": "2.6.6",
+    "jquery": "3.4.1",
     "lodash": "^4.17.15",
     "vue": "^2.6.10",
-    "core-js": "2.6.6",
-    "jquery": "3.4.1"
+    "file-saver": "2.0.2",
+    "node-sass": "4.13.0",
```
Comment (tomer-epstein) [inline frontend/package.json]: not used!

File: frontend/package.json [hunk 3]
```diff
@@ -31,15 +36,23 @@
     "babel-eslint": "^10.0.1",
     "babel-jest": "^24.9.0",
     "canvas": "2.5.0",
+    "deepmerge": "^4.2.2",
```
Comment (tomer-epstein) [inline frontend/package.json]: not used!

File: frontend/package.json [hunk 4]
```diff
@@ -31,15 +36,23 @@
...
+    "sass": "^1.24.2",
+    "sass-loader": "^8.0.0",
+    "lint-staged": "^8.1.5",
+    "husky": "1.2.1",
     "vue-jest": "^3.0.5",
     "vue-template-compiler": "^2.6.10",
     "webpack": "^4.41.2",
-    "flush-promises": "^1.0.2"
+    "vue-cli-plugin-vuetify": "^2.0.2",
+    "vuetify-loader": "^1.3.0"
```
Comment (tomer-epstein) [inline frontend/package.json]: not used!

File: frontend/src/App.vue [hunk 5]
```diff
@@ -8,58 +8,53 @@
...
-          </b-container>
-        </b-col>
-      </b-row>
-    </b-container>
-    <div>
-      <div :class="consoleClass">
-        <b-collapse id="showLogId" style="height: 100px; overflow-y: auto;" class="mt-auto">
-          <hr />
+    <v-row class="m-0 p-0">
+      <v-col v-if="prompts.length > 0" class="navigaiton-row m-0 p-0" lg="3" sm="auto">
```
Comment (tomer-epstein) [inline frontend/src/App.vue]: better place the ' v-if' on the 'Navigation' object

File: frontend/src/components/Navigation.vue [hunk 6]
```diff
@@ -1,49 +1,105 @@
...
+}
+.v-stepper__step--inactive .v-stepper__step__step {
+  background: var(--vscode-editorCodeLens-foreground, #1d1d1d);
+}
+
+/* TODO Erez to give styles for completed step */
+div.v-stepper
+  div.v-stepper__step.v-stepper__step--complete
+  div.v-stepper__label {
+  color: var(--vscode-editorGroup-border, #383838);
```
Comment (tomer-epstein) [inline frontend/src/components/Navigation.vue]: --vscode-editorGroup not supported in Theai yet. we should use other css var.

File: frontend/src/components/Navigation.vue [hunk 7]
```diff
@@ -1,49 +1,105 @@
...
+}
+div.v-stepper div.v-stepper__step.v-stepper__step--active div.v-stepper__label {
+  color: var(--vscode-settings-checkboxForeground, #f0f0f0);
```
Comment (tomer-epstein) [inline frontend/src/components/Navigation.vue]: not supported in Theai yet. we should use other css var.

File: frontend/src/components/Navigation.vue [hunk 8]
```diff
@@ -1,49 +1,105 @@
...
+}
+div.v-stepper
+  div.v-stepper__step.v-stepper__step--inactive
+  div.v-stepper__label {
+  color: var(--vscode-editorGroup-border, #383838);
```
Comment (tomer-epstein) [inline frontend/src/components/Navigation.vue]: vscode-editorGroup not supported in Theia yet.

**Comment by DoroNahari** [comment]
I already had conflict and resolved them and you maybe inserted a new code.

**Comment by slavik-lvovsky** [comment]
> I already had conflicts and resolved them and you maybe inserted a new code.

I know, sorry, but we had to do it because of critical bugs.
Please do it again. Now your change got the high priority and we will not add any new changes until you finish it.

File: frontend/public/index.html [hunk 9]
```diff
@@ -5,6 +5,8 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@mdi/font@latest/css/materialdesignicons.min.css">
```
Comment (tomer-epstein) [inline frontend/public/index.html]: you have added mdi. are those needed now?

Comment (DoroNahari) [reply frontend/public/index.html]: I forgot to delete them.

**Pull request merged**